svelte-kit-scss/src/lib/components/IssueSearch/IssueSearch.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { describe, it, expect } from 'vitest';
+import IssueSearchList from './IssueSearchList.svelte';
+import { render, screen } from '@testing-library/svelte';
+import { MOCK_ISSUE_ARRAY } from '$lib/helpers/mocks/issues';
+
+describe('IssueSearch', () => {
+  beforeEach(() => {
+    render(IssueSearchList, {
+      items: MOCK_ISSUE_ARRAY,
+    });
+  });
+
+  describe('List issues', () => {
+    it('should list all issues', () => {
+      const issues = screen.queryAllByTestId('issue-search-list-item');
+      expect(issues.length).toBe(4);
```

Comment:
We could extract the 4 and make it a const at the top of the file. Maybe even base it on the length of mock issues. This way the tests won't fail whenever the number of issues change.

Reply:
Makes sense!

Reply:
Fixed